src-tauri/src/aux_functions/mod.rs

```diff
@@ -0,0 +1 @@
+pub(crate) mod edge_factory;
```

Comment:
What is the `(create)` symbol out of curiosity?

Reply:
Not super sure vscode recommended that I put that.